src/test/java/pl/javagame/TestClass.java

```diff
@@ -0,0 +1,24 @@
+package pl.javagame;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestClass {
+    @Test
+    public void alwaysPass() {
+        assertTrue(true);
+    }
+
+    @Test
+    public void PlayerTestNull() {
+        Player p1 = new Player(null);
```

Comment:
Testy do porawy, potrzebują asercji. Jak poprawisz klase `Player` to zobacz jak przetestować czy metoda rzuciła wyjątek,